sandbox/readaligner_pairhmm_train.py

```diff
@@ -37,6 +36,8 @@ def main():
 
     args = parser.parse_args()
 
+    import pysamFOO
```

Comment:
I thought we found a Python only sam parser? All I can come up with is
https://pypi.python.org/pypi/libsam/ which doesn't fill me with confidence

On Tue, Jul 28, 2015 at 11:00 AM C. Titus Brown <notifications@github.com>
wrote:

> In sandbox/readaligner_pairhmm_train.py
> <https://github.com/dib-lab/khmer/pull/1121#discussion_r35678960>:
>
> > @@ -37,6 +36,8 @@ def main():
> >
> >      args = parser.parse_args()
> >
> > +    import pysamFOO
>
> Hah, I guess that's the problem with allowing it to be skipped!
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dib-lab/khmer/pull/1121/files#r35678960>.
>
-- 
Michael R. Crusoe: Programmer & Bioinformatician crusoe@ucdavis.edu
The lab for Data Intensive Biology; University of California, Davis
https://impactstory.org/MichaelRCrusoe http://twitter.com/biocrusoe